src/app/profile/overview/spaces/spaces.component.ts

```diff
@@ -76,31 +76,6 @@ export class SpacesComponent implements OnDestroy, OnInit  {
     }
   }
 
-  removeSpace(): void {
-    if (this.context && this.context.user && this.spaceToDelete) {
-      let space = this.spaceToDelete;
-      this.subscriptions.push(this.spaceService.deleteSpace(space)
-        .subscribe(spaces => {
-            let index = this.spaces.indexOf(space);
-            this.spaces.splice(index, 1);
-            this.spaceToDelete = undefined;
-            this.modalRef.hide();
-          },
-          err => {
-            this.logger.error(err);
-            this.spaceToDelete = undefined;
-            this.modalRef.hide();
-          }));
-    } else {
-      this.logger.error('Failed to retrieve list of spaces owned by user');
-    }
-  }
-
-  confirmDeleteSpace(space: Space, deleteSpace: TemplateRef<any>): void {
-    this.spaceToDelete = space;
-    this.modalRef = this.modalService.show(deleteSpace, {class: 'modal-lg'});
-  }
-
```

Comment:
It's unfortunate that users cannot delete their own spaces from this list. This means a user has to navigate away from this page and must "view all spaces".

Reply:
Agreed, but that will be in future implementations. This is just the simplified version for the time being.

Reply:
If the intention is to use this in the future, then I suggest leaving the code or comment it out for now. It took someone a bit of time to figure out how to delete a space.

Reply:
added back, but commented out